docs/webpack.config.dev.js

```diff
@@ -36,33 +35,39 @@ module.exports = {
     })
   ],
   module: {
-    loaders: [
+    rules: [
       {
         test: /\.json$/,
-        loaders: ['json-loader'],
+        use: ['json-loader'],
       },
       {
         test: /\.jsx?/,
-        loaders: ['babel'],
+        use: ['babel-loader'],
         include: [
           path.join(__dirname, 'src'),
           path.resolve(__dirname, './node_modules/linode-components'),
         ]
       },
       {
         test: /\.scss$/,
-        loaders: ['style', 'css', 'sass'],
+        use: [
+          'style-loader',
+          'css-loader',
+          {
+            loader: 'sass-loader',
+            options: {
+              includePaths: [
+                path.resolve(__dirname, './node_modules/bootstrap/scss/')
+              ]
+            }
+          }
+        ],
       },
       {
         test: /\.svg$/,
-        loaders: ['file'],
+        loader: ['file-loader'],
         include: path.join(__dirname, 'node_modules')
```

Comment:
are we are keeping this here and yanking it in the root `webpack.config.dev.js`?